OpenRA.Mods.Common/Traits/Armament.cs

```diff
@@ -20,6 +20,7 @@ namespace OpenRA.Mods.Common.Traits
 	public class Barrel
 	{
 		public WVec Offset;
+		public WVec OverlayOffset;
```

Comment:
```suggestion
		public WVec MuzzleOverlayOffset;
```
so we don't have issues if someone decides to add another type of overlay in the future.